src/components/augmint-ui/header/styles.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+const BaseHeader = `
+    color: ${theme.colors.white};
+`;
+
+export const StyledHeaderH3 = styled.h3`
+    ${BaseHeader};
```

Comment:
where are all the other styles for h3 coming from? if the appearance of this element is still depending on external stylesheet we should port those styles to the scoped style.js.